File: CAPITULO 01/Java/ex11.java
import java.io.*;
import java.util.*;
import java.math.*;
import java.text.*;

public class ex11{
	public static void main(String[]args){
		int num;
		double raiz,raizc,quadrado,cubo;
		Scanner dado;
		 dado = new Scanner (System.in);
		System.out.println("Digite um numero inteiro e positivo: ");
		num = dado.nextInt();
		quadrado = num*num;
		cubo = num*num*num;
		raiz = Math.sqrt((double) num);
		raizc = Math.cbrt ((double) num);
		DecimalFormat dc;
		dc = new DecimalFormat("0.00");
		System.out.println("O numero "+num+" ao quadrado é "+dc.format(quadrado)+
			"\nO numero "+num+" ao cubo é "+dc.format(cubo)+
				"\nA raiz quadrada do  numero "+num+" é " +dc.format(raiz)+
					"\nA raiz cubica do numero "+num+" é "+dc.format(raizc));
		
		
	}
}

File: CAPITULO 01/C++/ex11.cpp
#include <stdio.h>
#include <math.h>
int main()
{
	
	float num,quad,cubo,raiz,raiz_cub;
	printf("Digite um numero positivo real e inteiro : ");
	scanf("%f*c",&num);
	printf("\n");
	quad = num*num;
	printf("A numero %.2 ao quadrado eh %.2",num,quad);
	cubo = num*num*num;
	printf("\n");
	printf("A raiz cubica do numero %.2 eh %.2",num,cubo);
	raiz = sqrt(num);
	printf("\n");
	printf("A raiz quadrada do numero %.2f e %.2f",num,raiz);
	printf("\n");
	raiz_cub = cbrt(num);
	printf("A raiz cubica de %.2 eh = %.2",num,raiz_cub);
	getchar();s
	return 0;
}
